### PR TITLE
chore: Update after HubPool Hub <-> Spoke token updates

### PR DESCRIPTION
**Pull request by pxrl**

### CODE REVIEW

File: src/dataworker/PoolRebalanceUtils.ts [hunk 1]
```diff
@@ -476,7 +476,7 @@ export function getWidestPossibleExpectedBlockRange(
   // filled during the challenge period.
   const latestPossibleBundleEndBlockNumbers = chainIdListForBundleEvaluationBlockNumbers.map(
     (chainId: number, index) =>
-      spokeClients[chainId] && Math.max(spokeClients[chainId].latestBlockSearched - endBlockBuffers[index], 0)
+      spokeClients[chainId] && Math.max(spokeClients[chainId].latestBlockNumber - endBlockBuffers[index], 0)
```
Comment (nicholaspai) [inline src/dataworker/PoolRebalanceUtils.ts:479]: why this change?

Comment (pxrl) [reply src/dataworker/PoolRebalanceUtils.ts:479]: This was currently necessary for it to build on my end w/ sdk-v2 locally linked, but it will ultimately be subject to whatever change is introduced by your PR :+1:

Comment (pxrl) [reply src/dataworker/PoolRebalanceUtils.ts]: fwiw it's not there anymore because your subsequent change to sdk-v2 is also included in this version bump.

**Review by nicholaspai — APPROVED** [review]
